src/dawn/IIR/IIRSerializer.cpp

```diff
@@ -0,0 +1,833 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
+//                          _
+//                         | |
+//                       __| | __ ___      ___ ___
+//                      / _` |/ _` \ \ /\ / / '_  |
+//                     | (_| | (_| |\ V  V /| | | |
+//                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+//
+//
+//  This file is distributed under the MIT License (MIT).
+//  See LICENSE.txt for details.
+//
+//===------------------------------------------------------------------------------------------===//
+#include "dawn/IIR/IIRSerializer.h"
+#include "dawn/IIR/IIR.pb.h"
+#include "dawn/IIR/IIRNodeIterator.h"
+#include "dawn/IIR/MultiStage.h"
+#include "dawn/IIR/StatementAccessesPair.h"
+#include "dawn/IIR/StencilInstantiation.h"
+#include "dawn/Optimizer/PassComputeStageExtents.h"
+#include "dawn/Optimizer/PassInlining.h"
+#include "dawn/SIR/ASTVisitor.h"
+#include "dawn/SIR/SIR.h"
+#include "dawn/Serialization/ASTSerializer.h"
+#include <fstream>
+#include <google/protobuf/util/json_util.h>
+#include <list>
```

Comment:
none of this list, tuple, stack seem to be needed

Reply:
fixed